content/en/dev-tut-dvm-withdraw.md

```diff
@@ -0,0 +1,31 @@
+---
+id: dev-tut-dvm-withdraw
+title: Withdraw from DVM address
+sidebar_label: Withdraw from DVM address
+---
+
+> The DVM address format is identical to the Ethereum address format
+
+With the DVM smart contract solution, users can easily use ethereum contracts in the Darwinia Network, as well as the
```

Comment:
use --> deploy?

Reply:
I prefer to `use` because there are many aspects to smart contract practice, and deployment is only one part of it.